src/dispatch/incident/service.py

```diff
@@ -263,49 +263,80 @@ def delete(*, db_session, incident_id: int):
     db_session.commit()
 
 
-def calculate_cost(incident_id: int, db_session: SessionLocal, incident_review=False):
-    """Calculates the incident cost."""
-    # we ge the incident
+def get_engagement_multiplier(participant_role: str):
+    """Returns an engagement multiplier for a given incident role."""
+    engagement_mappings = {
+        ParticipantRoleType.incident_commander: 1,
+        ParticipantRoleType.scribe: 0.75,
+        ParticipantRoleType.liaison: 0.75,
+        ParticipantRoleType.participant: 0.5,
+        ParticipantRoleType.reporter: 0.5,
+    }
+
+    return engagement_mappings.get(participant_role, [])
+
+
+def calculate_cost(incident_id: int, db_session: SessionLocal, incident_review=True):
+    """Calculates the cost of a given incident."""
     incident = get(db_session=db_session, incident_id=incident_id)
 
-    participants_active_hours = 0
+    participants_total_response_time_seconds = 0.0
```

Comment:
Does this need to be a float/decimal? Are you capturing milliseconds?

Reply:
No, not trying to capture milliseconds. The result of adjusting the time with the engagement multiplier could result in a `float`. I'll make it an `int` and cast the result to `int` as well.